version-control/git-windows.md

```diff
@@ -0,0 +1,27 @@
+## Install Git on Windows
+
+1. Go to the [Git Downloads](https://git-scm.com/downloads) page and select the Windows icon to begin downloading Git to your computer. 
```

Comment:
Should we have the attendees first verify they don't have Git already installed? (or we can force them to update by installing the new version, I'm good with either). Your choice. :)

Reply:
I like the idea of having them update no matter what. It may cause some confusion for those who understand Git at a deeper level but am thinking those are few and far between and we should cater directions toward beginners. :)